felix/dataplane/linux/int_dataplane.go

```diff
@@ -2286,11 +2394,11 @@ func (d *InternalDataplane) loopReportingStatus() {
 	}
 }
 
-// IptablesTable is a shim interface for iptables.Table.
-type IptablesTable interface {
-	UpdateChain(chain *iptables.Chain)
-	UpdateChains([]*iptables.Chain)
-	RemoveChains([]*iptables.Chain)
+// Table is a shim interface for generictables.Table.
+type Table interface {
```

Comment:
nit: What if we move this type now to generictables as say `RuleChains` and embed it in `Table`?